intake_bluesky/core.py

```diff
@@ -461,8 +463,13 @@ def read_partition(self, i):
                     self.filler('event', event)
                 except event_model.UnresolvableForeignKeyError as err:
                     datum_id = err.key
-                    datum = self._get_datum(datum_id=datum_id)
-                    resource_uid = datum['resource']
+
+                    if '/' in datum_id:
+                        resource_uid, datum_id = datum_id.split('/', 1)
```

Comment:
I think we should not redefine the variable `datum_id` here. It's not used again anyway, but for debugging purposes best not to confuse things. This can just be `resource_uid, _ = ...`.

Reply:
updated